src/effects/builtin/whitenoiseeffect.cpp

```diff
@@ -20,7 +20,7 @@ EffectManifestPointer WhiteNoiseEffect::getManifest() {
 
     // This is dry/wet parameter
     EffectManifestParameterPointer intensity = pManifest->addParameter();
-    intensity->setId("Dry/Wet");
+    intensity->setId("dry_wet");
```

Comment:
Can you move the `dry_wet` string to a named constant in an anonymous namespace at the top of the file? This way it doesn't need to be updated in two places if it ever changes. I intend to do this for the rest of the built in effects after #2618 is merged.

```
namespace {
constexpr QString dryWetParameterId = "dry_wet";
} // anonymous namespace
```

Reply:
> Can you move the `dry_wet` string to a named constant in an anonymous namespace at the top of the file? This way it doesn't need to be updated in two places if it ever changes. I intend to do this for the rest of the built in effects after #2618 is merged.
> 
> ```
> namespace {
> constexpr QString dryWetParameterId = "dry_wet";
> } // anonymous namespace
> ```
In the header file or .cpp?

Reply:
I don't think QString can be made constexpr. 
```cpp
namespace {
const QString dryWetParameterId = QStringLiteral("dry_wet");
} // anonymous namespace
```

Reply:
In the cpp file.